makefile

```diff
@@ -16,10 +16,15 @@ build: clear install
 
 dev: clear install
 	$(call log, "starting DEV server...")
-	@NODE_ENV=development node __server__/dev-server.js
+	@NODE_ENV=development node __server__/development.js
 
-start:
-	@NODE_ENV=production pm2 start __server__/index.js
+start: stop
+	@NODE_ENV=production pm2 start __server__/production.js
+	@echo '' # just a spacer
+	$(call log, "PROD server is running at http://localhost:9000")
```

Comment:
If we're logging here, we can kill the [callback in the production server](https://github.com/guardian/guui/blob/c0d602d351c4754c0ff77b588e26fedf7d0bea80/__server__/production.js#L29)

Reply:
oooh yeah - i never saw that in the console, hence the log, did i just miss it?

Reply:
Maybe. I think I prefer it in the makefile anyway.